srml/support/src/dispatch.rs

```diff
@@ -487,6 +487,7 @@ macro_rules! decl_module {
 		$vis:vis fn $name:ident ( root $(, $param:ident : $param_ty:ty )* ) { $( $impl:tt )* }
 	) => {
 		impl<$trait_instance: $trait_name> $module<$trait_instance> {
+			#[doc(hidden)]
```

Comment:
why is this hidden ? some can be public, balances module doesn't show transfer function anymore